nexus/Cargo.toml

```diff
@@ -36,16 +34,12 @@ ipnetwork.workspace = true
 lazy_static.workspace = true
 macaddr.workspace = true
 mime_guess.workspace = true
-newtype_derive.workspace = true
 # Not under "dev-dependencies"; these also need to be implemented for
 # integration tests.
 nexus-test-interface.workspace = true
 num-integer.workspace = true
 once_cell.workspace = true
 openssl.workspace = true
-openssl-sys.workspace = true
-openssl-probe.workspace = true
```

Comment:
@jmpesp other than "tests pass" is there a way confirm this doesn't break anything? I see these are transitive deps of samael, but Nexus doesn't appear to depend on them directly.

Reply:
The samael / SAML tests are pretty extensive - I'd be more worried about incompatible openssl versions (or libxmlsec1, like [here](https://github.com/oxidecomputer/garbage-compactor/pull/3)), something that I think we test by deploying to a helios machine during CI. I don't think there's anything an actual end-to-end SAML login would exercise over the unit tests though.

Reply:
Gotcha, I'll trust those end-to-end tests then!